release_dashboard/lib/widgets/conductor_status.dart

```diff
@@ -268,10 +273,17 @@ class RepoInfoExpansionState extends State<RepoInfoExpansion> {
                       decoration: const BoxDecoration(border: Border(top: BorderSide(color: Colors.grey))),
                       children: <Widget>[
                         Text('$repoElement:'),
-                        SelectableText(
-                            (widget.currentStatus[repoElement] == null || widget.currentStatus[repoElement] == '')
-                                ? 'Unknown'
-                                : widget.currentStatus[repoElement]! as String),
+                        repoElement == ConductorStatus.engineRepoElements[URLelements['engine luci dashboard']!] ||
```

Comment:
Whenever conditionals start to wrap across several lines, it's cleaner to move them to a helper function. So something like `_isClickable` here